sections/highlights.html

```diff
@@ -0,0 +1,17 @@
+<section id="highlights">
+  <h1>Highlights</h1>
+  <div class="highlight">
+      <h2>Cnidario</h2>
+      <video controls width="500">
+          <source src="videos/cnidas1.mp4" type="video/mp4">
+          Your browser does not support the video tag.
+      </video>
+  </div>
+  <div class="highlight">
+      <h2>KickFlip</h2>
+      <video controls width="500">
+          <source src="videos/kickflip.mp4" type="video/mp4">
```

Comment:
Wdyt about uploading the video to Youtube and adding it embedded here?

Reply:
Done